app/models/analytics/AnalyticsService.scala

```diff
@@ -43,10 +43,19 @@ class AnalyticsService @Inject()(rpc: RPC, wkConf: WkConf, analyticsLookUpServic
   }
 }
 
-class AnalyticsLookUpService @Inject()(userDAO: UserDAO, wkConf: WkConf) extends LazyLogging {
+class AnalyticsLookUpService @Inject()(userDAO: UserDAO, multiUserDAO: MultiUserDAO, wkConf: WkConf)
+    extends LazyLogging {
+  implicit val ctx: DBAccessContext = GlobalAccessContext
+
+  def isSuperUser(userId: ObjectId): Fox[Boolean] =
+    for {
+      user <- userDAO.findOne(userId)
```

Comment:
Do we really need to take the detour with the `userId`?
I think we can directly pass the `multiUserId` and save one db call.

Reply:
Absolutely right! Good catch :) Could you have a look again?